services/beaconchain/src/main/java/tech/pegasys/teku/services/beaconchain/BeaconChainController.java

```diff
@@ -654,6 +660,11 @@ protected void initProposerSlashingPool() {
             beaconBlockSchemaSupplier.andThen(BeaconBlockBodySchema::getProposerSlashingsSchema),
             validator);
     blockImporter.subscribeToVerifiedBlockProposerSlashings(proposerSlashingPool::removeAll);
+    final ValidatorTimingChannel validatorTimingChannel =
+        eventChannels.getPublisher(ValidatorTimingChannel.class);
+    proposerSlashingPool.subscribeOperationAdded(
+        (operation, validationStatus, fromNetwork) ->
+            validatorTimingChannel.onProposerSlashing(operation));
```

Comment:
same re. being only relevant for this feature i believe..

Reply:
Same, fixed here [7918](https://github.com/Consensys/teku/pull/7918/files#diff-a0738cefef0ee011967c18c1fe03e9c169ab10cfc7a1e7531f31cd71505b7a39)